requirements.txt

```diff
@@ -44,11 +44,11 @@ graphql-core==2.3.2       # via django-graphql-jwt, django-ilmoitin, graphene, g
 graphql-relay==2.0.1      # via django-ilmoitin, graphene
 idna==2.10                # via requests
 itypes==1.2.0             # via coreapi
-jinja2==2.11.2            # via coreschema, django-ilmoitin, django-silk
+jinja2==2.11.3            # via coreschema, django-ilmoitin, django-silk
 json-log-formatter==0.3.0  # via -r requirements.in
 lockfile==0.12.2          # via django-mailer
 markupsafe==1.1.1         # via jinja2
-pillow==7.2.0             # via -r requirements.in
+pillow==8.1.2             # via -r requirements.in
```

Comment:
It seems like we're not even using it anymore 🤔

Reply:
All the better then, I'll remove it completely

Reply:
fixed